tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Extensions/HttpRequestExtensionsTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
```

Comment:
I think we can remove namespace `System.Threading.Tasks` in not needed.

Reply:
removed